docs/ucr/mobile_ucr_v2_migration_guide.rst

```diff
@@ -0,0 +1,88 @@
+Migrating to Mobile UCR Restore V2
+==================================
+
+Evolution of Mobile UCR Restore Versions
+----------------------------------------
+Historically, there were three configuration settings for how Mobile UCRs were sent to devices:
```

Comment:
nit: just to make this more user friendly you could say something like:

V1.0 was released where "all reports were bundled into a single fixture during the restore process".
V2.0 was released to enhance performance where each report was a fixture of its own.

In the interim, V1.5 was released to support both versions 1.0 and 2.0 together, while projects migrate of from version 1.0.